spec/abstract/AARCH64/Arch_Structs_A.thy

```diff
@@ -15,6 +15,10 @@ imports
   ExecSpec.Kernel_Config_Lemmas
 begin
 
+context begin interpretation Arch .
+lemmas [code] = pageBits_def ipa_size_def
```

Comment:
This is a bit of a workaround for the code generator. Usually, all equation definitions are made available to the code generator without explicit declaration. In locales, because they could have assumptions, this does not happen automatically, so I do that explicitly here. This is so that the `value_type` command later can use these values (because it uses the code generator to normalise the value to a number).